test/controllers/processors/InternalEmailFormSubmissionProcessor.ts

```diff
@@ -0,0 +1,91 @@
+import 'reflect-metadata'
+
+import { Arg } from '@fluffy-spoon/substitute';
+import * as assert from 'assert';
+import { Maybe, Session } from 'ch-node-session-handler';
+import { SessionKey } from 'ch-node-session-handler/lib/session/keys/SessionKey';
+import { SignInInfoKeys } from 'ch-node-session-handler/lib/session/keys/SignInInfoKeys';
+import { ISignInInfo, IUserProfile } from 'ch-node-session-handler/lib/session/model/SessionInterfaces';
+import { Request } from 'express';
+
+import { InternalEmailFormSubmissionProcessor } from 'app/controllers/processors/InternalEmailFormSubmissionProcessor';
+import { Appeal, APPEALS_KEY } from 'app/models/Appeal';
+import { EmailService } from 'app/modules/email-publisher/EmailService';
+import { loadEnvironmentVariablesFromFiles } from 'app/utils/ConfigLoader';
+
+import { createSubstituteOf } from 'test/SubstituteFactory';
+
+describe('InternalEmailFormSubmissionProcessor', () => {
```

Comment:
Shouldnt this be a .test.ts file rather than a .ts?

Reply:
Very good spot. Top class. Without these tests did not run via CLI.